src/com/xilinx/rapidwright/timing/TimingModel.java

```diff
@@ -500,17 +502,82 @@ protected List<TimingGroup> determineGroups(List<Node> nodes, List<IntentCode> n
         return result;
     }
 
+    private Pair<Integer,Integer> findReferenceTileLocation(Device d) {
+        class Helper {
+            Map<Integer, Set<Integer>> spans;
+            public Helper() {
+                spans = new HashMap<Integer,Set<Integer>>();
+            }
+            void insert(int val, int span) {
+                if (!spans.containsKey(span)) {
+                    spans.put(span, new HashSet<Integer>());
+                }
+                spans.get(span).add(val);
+            }
+            Set<Integer> longest() {
+                List<Integer> keys = new ArrayList<>(spans.keySet());
+                Integer maxSpan = Collections.max(keys);
+                return spans.get(maxSpan);
+            }
+        }
+
+
+        // for each column, look for valid row
+        Helper colHelper = new Helper();
+        for (int i = 0; i < d.getColumns(); i++) {
+            int span = 0;
+            for (int j = 0; j < d.getRows(); j++) {
+                if (d.getTile("INT_X" + i + "Y" + j) != null) {
+                    span++;
+                }
+            }
+            colHelper.insert(i,span);
+        }
+
+        // for each row, look for valid col
+        Helper rowHelper = new Helper();
+        for (int j = 0; j < d.getRows(); j++) {
+            int span = 0;
+            for (int i = 0; i < d.getColumns(); i++) {
+                if (d.getTile("INT_X" + i + "Y" + j) != null) {
+                    span++;
+                }
+            }
+            rowHelper.insert(j,span);
+        }
+
+        for (int INTCol : colHelper.longest()) {
+            for (int INTRow : rowHelper.longest()) {
+                Tile tile = d.getTile("INT_X" + INTCol + "Y" + INTRow);
+                int col = tile.getColumn();
+                int row = tile .getRow();
+                // Want an INT tile that has CLB on both side
+                if (Utils.isCLB(d.getTile(row, col-1).getTileTypeEnum()) && Utils.isCLB(d.getTile(row, col+1).getTileTypeEnum())) {
+                    return new Pair<>(col,row);
+                }
+            }
+        }
+        return new Pair<>(-1,-1);
+    }
+
+
     /**
      * Reads the text file containing the delay terms needed by this timing model.
      * @param filename Name (and maybe the path) of the text file, the default is delay_terms.dat in the current directory.
      * @return Boolean indication of completion.
      */
     protected boolean readDelayTerms(String filename) {
-        boolean result = true; 
-        try (BufferedReader br = new BufferedReader(new FileReader(FileTools.getRapidWrightPath() + File.separator + filename))){
+
+        // Compute before reading from file to allow overriding.
+        Pair<Integer,Integer> loc = findReferenceTileLocation(device);
+        START_TILE_COL = loc.getFirst();
+        START_TILE_ROW = loc.getSecond();

```

Comment:
These variable names are neither static nor final.  They should probably use a lowerCamelCase naming.

Reply:
These are just two of like 20 legacy variable names.  If we change these two, we have to change them all.  Are you OK with that?

Reply:
Actually, there are like 80 of them!

Reply:
Let's defer then to another PR for now.